# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderSharedSurfaceTextureHostSWGL.cpp
# Commit: 9d637c05af7f
# Full Hash: 9d637c05af7f8c39af78b290335ca55994849ee8
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-06 04:14:34
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add RenderTextureHostSWGL support to D3D11 RenderTextureHosts. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95799
# ==============================================================================

diff -r ea88b243717a -r 9d637c05af7f gfx/webrender_bindings/RenderSharedSurfaceTextureHostSWGL.cpp
--- a/gfx/webrender_bindings/RenderSharedSurfaceTextureHostSWGL.cpp	Fri Nov 06 00:00:47 2020 +0000
+++ b/gfx/webrender_bindings/RenderSharedSurfaceTextureHostSWGL.cpp	Fri Nov 06 00:01:36 2020 +0000
@@ -34,7 +34,8 @@
   return gfx::ColorDepth::COLOR_8;
 }
 
-bool RenderSharedSurfaceTextureHostSWGL::MapPlane(uint8_t aChannelIndex,
+bool RenderSharedSurfaceTextureHostSWGL::MapPlane(RenderCompositor* aCompositor,
+                                                  uint8_t aChannelIndex,
                                                   PlaneInfo& aPlaneInfo) {
   if (NS_WARN_IF(
           !mSurface->Map(gfx::DataSourceSurface::MapType::READ, &mMap))) {